### ISSUE TITLE
Shuttingdown the Keyboard

### ISSUE THREAD

**Issue by mehalter** [makerdiary/python-keyboard#16]
I seem to be having issues shutting the keyboard down both when I press the ON/OFF button and then I do the SHUTDOWN keycode. It shuts down for a second and then comes back on. Is there anyway to be able to shutdown the keyboard entirely until I press the ON/OFF button again?

**Comment by xiongyihui** [comment]
Do you have USB connection?
When USB is connected, it will not shutdown. 
When USB is disconnected, holding down the ON/OFF for more than 3 seconds will shutdown the keyboard. <kbd>Fn</kbd> + <kbd>b</kbd> will also shutdown the keyboard.

**Comment by mehalter** [comment]
Yeah I have been trying this as well, but it is still turning on immediately after. The suspend function seems to work as intended tho

**Comment by xiongyihui** [comment]
That's weird. When the keyboard is powered by the battery, any reset should power off the keyboard. 
With the default keymap, The `BOOT` keycode (<kbd>Fn</kbd> + <kbd>b</kbd>) will reset the keyboard, and only run the built-in bootloader if powered by USB. If powered by the battery, the keyboard will power off before getting a chance to run the bootloader.
